hal/lib/athena/handles/IndexedHandleResource.h

```diff
@@ -16,6 +16,9 @@
 #include "HAL/cpp/priority_mutex.h"
 #include "HandlesInternal.h"
 
+#define INDEXED_RESOURCE_INDEX_OUT_OF_RANGE -1
```

Comment:
Good point. I forgot this was an internal class and couldn't be accessed from the C api. I'll fix it.